cli/src/cli_util.rs

```diff
@@ -2697,14 +2697,19 @@ pub fn print_snapshot_stats(
         })
         .max()
     {
+        let large_files = stats
+            .untracked_paths
+            .keys()
+            .map(|p| path_converter.format_file_path(p))
+            .join(" ");
```

Comment:
Nit: I think the `jj file track` hint should only be showed when `snapshot.autotrack != "all()"` as otherwise `jj st` still is applicable.

Reply:
I don't think we should get a `FileTooLarge` for paths that are also `FileNotAutoTracked`, so if a path is too large, it means the file will be auto-tracked.

Reply:
Wait, I was confused. The whole reason for the bug report and this patch is that we display this hint also when you are not auto-tracking all paths. This hint gets printed because we temporarily try to snapshot just the specified files. Maybe we should print a different message in `src/commands/file/track.rs` instead?

Reply:
Should replace the call to `print_snapshot_stats` with custom logic in `track.rs`? Or pass a new argument to `print_snapshot_stats` to specify when it gets called from `track`?

Reply:
Good question. I was thinking of replacing it, but it looks like most of it is relevant also for `jj file track`. Passing a flag is a bit ugly but it seems good enough to me. What do you think?

Reply:
I've added a flag to `print_snapshot_stats`. I'm not really satisfied with how it looks, most of the hint message is duplicated, maybe I should print it in several parts, gated by the flag?